async & performance/ch1.md

```diff
@@ -501,11 +501,11 @@ function answer(data) {
 }
 
 // ajax(..) é uma função Ajax eventual fornecida por uma biblioteca
-ajax( "http://some.url.1", answer );
-ajax( "http://some.url.2", answer );
+ajax( "http://some.url.1", response );
+ajax( "http://some.url.2", response );
```

Comment:
oi @gustavofabro, não traduzimos trechos de código (a menos que sejam comentários). Pode reverter esses trechos?

Reply:
Opa @cezaraugusto. Mas esta parte no inglês está `response` mesmo, e não `answer`. No caso não fiz a tradução, só mantive igual.

Reply:
Oi Gustavo, você está guiando a tradução pelo `upstream` ou pelo original daqui do repo (branch `master`)? Os dois livros (tradução e o repo original) tem diferença de versões.

Reply:
Olá @cezaraugusto, me guiei pela versão em inglês do branch master. Pelo que conferi no histórico, na primeira tradução foi [traduzido](https://github.com/cezaraugusto/You-Dont-Know-JS/commit/2f22aca0fc4855bed744bdaeed0c9f293a8de96c) de `response` para `resposta` e depois foi [voltado](https://github.com/cezaraugusto/You-Dont-Know-JS/blob/573fc545583950e701be3e177ffae4cf9179935a/async%20&%20performance/ch1.md#L505) para o inglês, porém como `answer`.

Reply:
Obrigado por investigar, você tem razão. Vou aprovar o envio.